### PR TITLE
t411 v2

### PR DESCRIPTION
**Pull request by ploufs**

### CODE REVIEW

**Comment by kaso17** [comment]
If tracker are deleted the definition file must be added here:
https://github.com/Jackett/Jackett/blob/master/src/Jackett.Updater/Program.cs#L180-L190
Without it, it won't be deleted during the update. Can you please add maniatorrent there?

File: src/Jackett.Updater/Program.cs [hunk 1]
```diff
@@ -185,6 +185,7 @@ private void ProcessUpdate(UpdaterConsoleOptions options)
                 "Definitions/rarbg.yml",
                 "Definitions/t411.yml",
                 "Definitions/hdbc.yml", // renamed to hdbitscom
+				"Definitions/maniatorrent.yml",
```
Comment (kaso17) [inline src/Jackett.Updater/Program.cs]: please fix the formatting, thank you!

**Comment by hed0nist** [comment]
Can you confirm the nextorrent definition working fine in its actual state ? Because even with a correct url it doesn't pass internal test and can't retrieve torrent files.
I made a working version for my personnal use and wanted confirmation of non-working state before trying to pull corrections.

**Comment by flightlevel** [comment]
Thank you

**Pull request merged**